test/javascript/components/Editor.test.js

```diff
@@ -7,67 +7,6 @@ import { rest } from 'msw'
 import { setupServer } from 'msw/node'
 import { Editor } from '../../../app/javascript/components/Editor'
 
-test('clears current submission when resubmitting', async () => {
```

Comment:
This test shouldn't apply anymore as we already cancel requests just in case a user submits two iterations at the same time.